### PR TITLE
[Doc] Bugfix in routine load

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: docs/sql-reference/sql-statements/data-manipulation/CREATE ROUTINE LOAD.md [hunk 1]
```diff
@@ -135,8 +135,9 @@ The properties of the data source.
...
+| kafka_partitions  | No       | The Kafka partitions to be consumed, for example, `"kafka_partitions" = "0, 1, 2, 3"`. If this property is not specified, all partitions are consumed by default. |
+| kafka_offsets     | No       | The starting offset from which to consume data in a Kafka partition as specified in `kafka_partitions`. If this property is not specified, the Routine Load job consumes data starting from the latest offsets for partitions in `kafka_partitions`. Valid values:<ul><li>A specific offset: consumes data starting from a specific offset.</li><li>`OFFSET_BEGINNING`: consumes data starting from the earliest offset possible.</li><li>`OFFSET_END`:  consumes data starting from the latest offset.</li></ul> Multiple starting offsets are separated by commas (,). |
+| property.kafka_default_offsets| No| The default starting offset for all consumer partitions. The supported values for this property are same as there for the `kafka_offsets` property.|
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/CREATE ROUTINE LOAD.md:140]: there --- those

File: docs/sql-reference/sql-statements/data-manipulation/CREATE ROUTINE LOAD.md [hunk 2]
```diff
@@ -135,8 +135,9 @@ The properties of the data source.
...
+| kafka_partitions  | No       | The Kafka partitions to be consumed, for example, `"kafka_partitions" = "0, 1, 2, 3"`. If this property is not specified, all partitions are consumed by default. |
+| kafka_offsets     | No       | The starting offset from which to consume data in a Kafka partition as specified in `kafka_partitions`. If this property is not specified, the Routine Load job consumes data starting from the latest offsets for partitions in `kafka_partitions`. Valid values:<ul><li>A specific offset: consumes data starting from a specific offset.</li><li>`OFFSET_BEGINNING`: consumes data starting from the earliest offset possible.</li><li>`OFFSET_END`:  consumes data starting from the latest offset.</li></ul> Multiple starting offsets are separated by commas (,). |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/CREATE ROUTINE LOAD.md:139]: 1. from the latest offsets for partitions in `kafka_partitions` ----  delete "for xxx" part for conciseness  2. `OFFSET_END`:  consumes --- extra space

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

**Comment by hellolilyliuyi** [comment]
@mergify backport main

**Pull request merged**